src/frontend/apps/desk/src/features/teams/components/ModalUpdateTeam.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Button,
+  Input,
+  Loader,
+  Modal,
+  ModalSize,
+  VariantType,
+  useToastProvider,
+} from '@openfun/cunningham-react';
+import { t } from 'i18next';
+import { useState } from 'react';
+
+import { Box, Text, TextErrors } from '@/components';
+import useCunninghamTheme from '@/cunningham/useCunninghamTheme';
+
+import { Team, useUpdateTeam } from '../api';
+import IconEdit from '../assets/icon-edit.svg';
+
+interface ModalUpdateTeamProps {
+  onClose: () => void;
+  team: Team;
+}
+
+export const ModalUpdateTeam = ({ onClose, team }: ModalUpdateTeamProps) => {
+  const { colorsTokens } = useCunninghamTheme();
+  const [newTeamName, setNewTeamName] = useState('');
+  const { toast } = useToastProvider();
+  const {
+    mutate: updateTeam,
+    isError,
+    isPending,
+    error,
+  } = useUpdateTeam({
+    onSuccess: () => {
+      toast(t('The team has been updated.'), VariantType.SUCCESS, {
+        duration: 4000,
+      });
+      onClose();
+    },
+  });
+
+  return (
+    <Modal
+      isOpen
+      closeOnClickOutside
+      hideCloseButton
+      leftActions={
+        <Button
+          aria-label={t('Close the modal')}
+          color="secondary"
+          fullWidth
+          onClick={() => onClose()}
+        >
+          {t('Cancel')}
+        </Button>
+      }
+      onClose={() => onClose()}
+      rightActions={
+        <Button
+          aria-label={t('Validate the modification')}
+          color="primary"
+          fullWidth
+          onClick={() =>
+            updateTeam({
+              name: newTeamName,
+              id: team.id,
+            })
+          }
+        >
+          {t('Validate')}
+        </Button>
+      }
+      size={ModalSize.MEDIUM}
+      title={
+        <Box $align="center" $gap="1rem">
+          <IconEdit width={48} color={colorsTokens()['primary-text']} />
+          <Text $size="h3" className="m-0">
+            {t('Update team {{teamName}}', { teamName: team.name })}
+          </Text>
+        </Box>
+      }
+    >
+      <Box className="mb-xl" aria-label={t('Content modal to update the team')}>
+        <Text as="p" className="mb-b">
+          {t('Enter the new team name')}
+        </Text>
+
+        <Input
```

Comment:
Should the input be in error status mode until the user types a new caracter?
<img width="668" alt="Capture d’écran 2024-03-21 à 15 29 41" src="https://github.com/numerique-gouv/people/assets/45729124/e8165e26-5d99-4c5a-a8eb-84913d35cb20">

Reply:
btw, I know you followed Johann's design. What do you think of populating the input with the current Team's name. Most of the time, people might fix typo or small error in their team's name. It would be so nice if they don't have to type again the full name. It's clearly a nice to have feature, which can be implementer later on 🤩

Reply:
> btw, I know you followed Johann's design. What do you think of populating the input with the current Team's name. Most of the time, people might fix typo or small error in their team's name. It would be so nice if they don't have to type again the full name. It's clearly a nice to have feature, which can be implementer later on 

https://github.com/numerique-gouv/people/pull/136/commits/127e73de62afd72abae026ee4f35d2b1476e1b09#diff-1ae3a2fb05049b93ef9e77297b00d499c33b85238d5bda17fc77e66acedeccb6R91

Reply:
> Should the input be in error status mode until the user types a new caracter?

https://github.com/numerique-gouv/people/pull/136/commits/127e73de62afd72abae026ee4f35d2b1476e1b09#diff-7e1f479914421dd6a51af79ca681712f5fc799962384a6b3f50e84ff0d0c7ed2R44
https://github.com/numerique-gouv/people/pull/136/commits/ae17b0036b9ec6f4cf2ba98dc750d935dc3cdd61